src/models/initgamemodel.js

```diff
@@ -5,8 +5,12 @@ class InitGameModel extends Model
      */
     setupCallback()
     {
+        var availableProducts = this.model.base.products.filter(product => {
```

Comment:
`const` or `let`. I remember we started off with `var`, but I've been moving away from those for quite some time now :).